vembrane/__init__.py

```diff
@@ -65,10 +67,10 @@ def filter_vcf(vcf: VariantFile, expression: str) -> Iterator[VariantRecord]:
         env["POS"] = record.pos
         env["REF"], env["ALT"] = chain(record.alleles)
         for key in record.info:
-            if key != "ANN":
+            if key != ann_key:
                 env[key] = record.info[key]
 
-        annotations = record.info.get("ANN", [])
+        annotations = dict(record.info).get(ann_key, [])
```

Comment:
Why does it crash? Looking at the [`VariantRecordInfo.get`](https://github.com/pysam-developers/pysam/blob/2282e99dde733360491cedfbfa6a8b6a4d191d6d/pysam/libcbcf.pyx#L2657) method, it should work without the dict wrapping

Reply:
it should, but it doesn't, give it a try

Reply:
on that note, make sure to add a test for that